gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/06_add_scoring_table.yml

```diff
@@ -6,6 +6,7 @@ databaseChangeLog:
         - createTable:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.